.kitchen.yml

```diff
@@ -1,6 +1,8 @@
 ---
 driver:
   name: vagrant
+  vagrantfiles: 
+      - vagrantfile.rb
```

Comment:
Just curious. Why the additional vagrantfile? You can add ruby to the regular `Vagrantfile`. Feel free to modify that one if you need to.